oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/remote/client/RemoteClientManager.java

```diff
@@ -55,21 +59,29 @@
     private static final Logger logger = LoggerFactory.getLogger(RemoteClientManager.class);
 
     private final ModuleDefineHolder moduleDefineHolder;
+    private final boolean grpcSslEnabled;
+    private final String grpcSslTrustCAPath;
     private ClusterNodesQuery clusterNodesQuery;
     private volatile List<RemoteClient> usingClients;
     private GaugeMetrics gauge;
     private int remoteTimeout;
 
     /**
      * Initial the manager for all remote communication clients.
-     *
-     * @param moduleDefineHolder for looking up other modules
+     *  @param moduleDefineHolder for looking up other modules
      * @param remoteTimeout      for cluster internal communication, in second unit.
+     * @param grpcSslEnabled true: enable SSL between clusters, false: plain gRPC between the cluster.
+     * @param grpcSslTrustCAPath trust certificate authorities file path.
      */
-    public RemoteClientManager(ModuleDefineHolder moduleDefineHolder, int remoteTimeout) {
+    public RemoteClientManager(ModuleDefineHolder moduleDefineHolder,
```

Comment:
Could you make the core style consistently. `GRPCRemoteClient` client use two constructors for ssl ON/OFF, here using a boolean and empty/null string.

Reply:
done. keep all parameter list the same way.